src/bcf-toolkit/Builder/Bcf30/BcfBuilder.cs

```diff
@@ -1,40 +1,46 @@
 using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Threading.Tasks;
-using BcfToolkit.Converter;
-using BcfToolkit.Model;
+using BcfToolkit.Builder.Bcf30.Interfaces;
+using BcfToolkit.Builder.Interfaces;
 using BcfToolkit.Model.Bcf30;
 using Bcf = BcfToolkit.Model.Bcf30.Bcf;
 using Markup = BcfToolkit.Model.Bcf30.Markup;
-using VisualizationInfo = BcfToolkit.Model.Bcf30.VisualizationInfo;
 
 namespace BcfToolkit.Builder.Bcf30;
 
 public partial class BcfBuilder : IBcfBuilder<
     BcfBuilder,
     MarkupBuilder,
-    ProjectBuilder>,
+    ProjectBuilder,
+    ExtensionsBuilder,
+    DocumentInfoBuilder>,
   IDefaultBuilder<BcfBuilder> {
-
   private readonly Bcf _bcf = new();
 
   public BcfBuilder AddMarkup(Action<MarkupBuilder> builder) {
     var markup =
-      (Markup)BuilderUtils.BuildItem<MarkupBuilder, IMarkup>(builder);
+      BuilderUtils.BuildItem<MarkupBuilder, Markup>(builder);
     _bcf.Markups.Add(markup);
     return this;
   }
 
-  public BcfBuilder AddMarkups(List<Markup> markups) {
-    markups.ForEach(m => _bcf.Markups.Add(m));
+  public BcfBuilder SetProject(Action<ProjectBuilder> builder) {
```

Comment:
Good catch, previously when there was a common interface for both versions, that made sense to implement one function, but know they are separated, i will change, thanks.

Reply:
However, i would use the `SetProject` because in the bcf the field name is `Project`
`_bcf.Project = projectInfo;`
And we would rename the builder instead
`var projectInfo = BuilderUtils.BuildItem<ProjectInfoBuilder, ProjectInfo>(builder);`
What do you think?

Reply:
I think thats fine too